x-pack/plugins/observability_solution/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/waterfall/index.tsx

```diff
@@ -48,8 +47,8 @@ const toggleFlyout = ({
   });
 };
 
-const WaterfallItemsContainer = euiStyled.div`
-  border-bottom: 1px solid ${({ theme }) => theme.eui.euiColorMediumShade};
+const WaterfallItemsContainer = styled.div`
+  border-bottom: 1px solid ${({ theme }) => theme.euiTheme.colors.mediumShade};
```

Comment:
We can make use of `euiTheme.border.width.thin` here for `1px` (same for usage further down)